docs/resources/group.md

```diff
@@ -30,6 +30,11 @@ resource "twingate_group" "aws" {
 
 - `name` (String) The name of the group
 
+### Optional
+
+- `authoritative` (Boolean) Determines authoritative behaviour for handling the Group's users.
```

Comment:
--> "Determines whether User assignments to this Group will override any existing assignments. Default is `true`. If set to `false`, assignments made outside of Terraform will be ignored."